Formula/awscurl.rb

```diff
@@ -0,0 +1,80 @@
+class Awscurl < Formula
+  include Language::Python::Virtualenv
+
+  desc "Curl like simplicity to access AWS resources"
+  homepage "https://github.com/okigan/awscurl"
+  url "https://github.com/okigan/awscurl/archive/v0.20.tar.gz"
+  version "v0.20"
```

Comment:
This is not needed with the tagged version in the URL

Reply:
fixed